### ISSUE TITLE
Unable to setup properly

### ISSUE THREAD

**Issue by Sandeep28633** [swensonhe/strapi-firebase-auth#64]
Cannot read properties of undefined (reading 'auth') at .../node_modules/@swensonhe/strapi-plugin-firebase-auth/server/services/userService.js:8:71

**Comment by tothang** [comment]
@Sandeep28633  Did you use firebaseConfig to set up ? If yes. Please download your serviceAccountKey.json copy all value from that file and paste to the input that will help to solve the issue

**Comment by Sandeep28633** [comment]
@tothang  yes Followed all the steps

**Comment by tothang** [comment]
> @tothang yes Followed all the steps

download your serviceAccountKey.json copy all value from that file and paste to the input instead inputing firebaseConfig. That will solve your issue. Check for how to download https://clemfournier.medium.com/how-to-get-my-firebase-service-account-key-file-f0ec97a21620

**Comment by Sandeep28633** [comment]
<img width="925" alt="image" src="https://github.com/user-attachments/assets/fc7965e9-4c5c-4127-b434-b48fa3c75621">
SHould I paste here. ?

**Comment by tothang** [comment]
@Sandeep28633 Yep. please paste all item in serviceAccountKey.json to that input and come back here with the result

**Comment by Sandeep28633** [comment]
that seems to be working. but after doing all these, when I am navigating to content manager, it giving me blank page
<img width="1346" alt="image" src="https://github.com/user-attachments/assets/f45bfc74-bd9f-40f1-bf72-521a942f4110">

**Comment by tothang** [comment]
@Sandeep28633  user need to login  by strapi for syncing with users table of strapi. That is correct behaviour . You cannot just add user on firebase console and expect that user getting synced with strapi users. If you really wana do that let''s write a script to synced data from strapi to your users tables

**Comment by tothang** [comment]
@Sandeep28633  go to strapi. Setting > Role > Public and look for auth firebase then enable. You can call api login to strapi by firebase token

**Comment by bhartendu27** [comment]
@Sandeep28633 
This issue is being caused by incorrect documentation, you will need the serviceAccount json file from your firebase console. After that COPY THE CONTENTS OF SERVICE ACCOUNT JSON FILE (NOT example firebase code, as shown in the docs) and paste it in firebase configuration inside strapi. Hope this helps.
PS: don't forget to enable the firebase authentication token validation route inside the public role
